packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,160 @@
+default.target=.
+
+## all templates to apply on target deployment env
+nuxeo.template.includes=default,s3binaries,postgresql,explorer-sitemode,explorer-virtualadmin
+# TODO: check logs config with 'docker' or 'docker-json' template
+
+## explorer confs mapping
+org.nuxeo.apidoc.apidocAdmin.password=${env:ORG_NUXEO_APIDOC_APIDOCADMIN_PASSWORD, ""}
+
+
+## nuxeo.conf mappings
+
+nuxeo.log.dir=${env:NUXEO_LOG_DIR:/var/log/nuxeo}
+nuxeo.pid.dir=${env:NUXEO_PID_DIR:/var/run}
+nuxeo.tmp.dir=${env:NUXEO_TMP_DIR:/tmp}
+
+
+## default confs mapping
+
+nuxeo.data.dir=${env:NUXEO_DATA_DIR:nxserver/data}
+nuxeo.server.dir=${env:NUXEO_SERVER_DIR:nxserver}
+
+# Default IP address binding
+nuxeo.bind.address=${env:NUXEO_BIND_ADDRESS:0.0.0.0}
```

Comment:
I'm not sure we need this as we're in docker.

Reply:
dunno, i guess we will see

Reply:
I believe that the behavior in the default docker image should be ok, you should not need to override this as part of your deployment.

Reply:
Removed